armi/utils/tests/test_densityTools.py

```diff
@@ -100,6 +107,13 @@ def test_applyIsotopicsMix(self):
         self.assertAlmostEqual(uo2.massFrac["O"], massFracO)  # non-HM stays unchanged
 
     def test_getMassFractions(self):
+        """
+        Number densities to mass fraction.
+
+        .. test:: Number densities to mass fractions
+            :id: T_ARMI_UTIL_MASS2N_DENS0
+            :tests: R_ARMI_UTIL_MASS2N_DENS
```

Comment:
Much like how above I said this req doesn't go with this function, I think this test (of this function) doesn't go with this req.

We can just remove this `..test::` tag here.